test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -96,5 +104,15 @@ Describe "Stream writer tests" -Tags "CI" {
             $result[0].MessageData | Should be "teststring"
             $result[1].MessageData | Should be "12345"
        }
+
+       It "Write-Information works with <Name>" -TestCases:$testInfoData {
+            param($Command, $returnCount, $returnValue)
+            $ps.AddScript($Command).Invoke()
+
+            $result = $ps.Streams.Information
+
+            $result.Count | Should Be $returnCount
+            (Compare-Object $result $returnValue -SyncWindow 0).length -eq 0 | Should Be $true
```

Comment:
This should be as follows for a better error message when it fails:

`(Compare-Object $result $returnValue -SyncWindow 0).length | Should Be 0`

Reply:
I like this suggestion. I made the same change in the Write-Host Tests too.